apps/projects/app/components/Content/Issues.js

```diff
@@ -487,7 +487,7 @@ const ScrollWrapper = styled.div`
 // TODO: Calculate height with flex (maybe to add pagination at bottom?)
 const IssuesScrollView = styled.div`
   height: 75vh;
-  min-width: 600px;
+  /*min-width: 600px;*/
```

Comment:
Remove instead of comment if not needed.

Reply:
Ok.

Reply:
This was not removed